app/models/user-model.coffee

```diff
@@ -32,7 +32,9 @@ global.cobudgetApp.factory 'UserModel', (BaseModel) ->
       @administeredGroups().length > 0
 
     primaryGroup: ->
-      @groups()[0]
+      groupsByLastActivity = _.sortBy @groups(), (group) ->
+        group.lastActivityAt
+      groupsByLastActivity[groupsByLastActivity.length - 1]
```

Comment:
IIUC it seems like this will return the group with the most recent activity by _anyone_, as opposed to the group last used by _this_ user.